### ISSUE TITLE
Status Bar toggle not working

### ISSUE THREAD

**Issue by nickwarters** [lynchjames/obsidian-day-planner#37]
Toggling the Status Bar setting on and off does nothing and the Status Bar remains disabled despite the toggle being enabled. 
No log in the console when toggling between enabled and disabled. 

Using Obsidian v0.9.19 and Day Planner v0.5.3

**Comment by lynchjames** [comment]
Do you mean the "Status Bar - Circular Progress" or "Status Bar - Now and Next" setting? 

A status bar should always appear if you have a day planner linked to a note with an active task. This means having a task that starts before the current time and has a following task starting after the current time. 

Do you have anything being displayed in the timeline view when you do not see a status bar being displayed?

Are you also using the Hider plugin?

**Comment by nickwarters** [comment]
Both elements of the status bar remain hidden/do not show in the status bar. 
I do see the timeline view perfectly fine, which is a great feature.

I’m not using the hider plugin no.

**Comment by wfischer42** [comment]
Same issue here. I'm using a couple other plugins, but not hider. It doesn't seem to be working in either mode.

**Comment by mikebarkmin** [comment]
I have the same problem. I checked see console and saw theses errors:
```
app.js:1 Obsidian Developer Console
main.ts:179 Loading Day Planner plugin
main.ts:232 loading markdown-table-editor plugin
main.ts:232 Found existing settings file
parser.ts:24 Error: ENOENT: no such file or directory, open 'home/Day Planners/Day Planner-20201203.md'
file.ts:25 TypeError: Cannot read property 'split' of undefined
    at PlannerMarkdown.parseDayPlanner (eval at <anonymous> (app.js:1), <anonymous>:6219:9)
    at async eval (eval at <anonymous> (app.js:1), <anonymous>:7532:33)
main.ts:179 TypeError: Cannot read property 'calculate' of undefined
    at eval (eval at <anonymous> (app.js:1), <anonymous>:7533:25)
```
But the file is at the correct location. Maybe the plugin is faster in trying to check if the file is there than in creating the file. Maybe this also hinders the Status Bar from popping up.

**Comment by nickwarters** [comment]
Update on my issue - If I enable to status bar, add in time entries to populate the chart, then quit and reopen Obsidian, the status bar works. It will not work for me unless I quit and reopen the app once I have populated things into the day plan.

**Comment by lynchjames** [comment]
Definitely seems like I have introduced a bug! @nickwarters and @wfischer42 are you seeing similar errors in the console to @mikebarkmin?

**Comment by nickwarters** [comment]
@lynchjames I'm not seeing any output in the console. 
Right this moment it is 07:16 and I have added 3 times into the plan: 06:00, 07:00, and 08:00.
The 06:00 is marked as completed, but not the 07:00, and the status bar shows "All Done".

**Comment by lynchjames** [comment]
I have found a bug with the File mode when a new note is first created meaning that the status bar would not be displayed. There will be a fix in the next release.

**Comment by lynchjames** [comment]
This should now be fixed in the latest release ([0.5.4](https://github.com/lynchjames/obsidian-day-planner/releases/tag/0.5.4)).
